app/src/main/java/org/oppia/android/app/recyclerview/BindableAdapter.kt

```diff
@@ -223,10 +202,11 @@ class BindableAdapter<T : Any> internal constructor(
           viewGroup,
           /* attachToRoot= */ false
         )
-        binding.lifecycleOwner = getLifecycleOwner()
+
         object : BindableViewHolder<T>(binding.root) {
           override fun bind(data: T) {
             setViewModel(binding, data)
+            binding.lifecycleOwner = getLifecycleOwner()
```

Comment:
Can you please add a comment here explaining why this has to happen here specifically (per the issues that you ran into)?

Reply:
Yeah sure thing.